packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -38,6 +38,12 @@ const DEBUG = mode !== production && process.env.DEBUG === 'true'
 const CI = process.env.CI
 const disableHMR = process.env.HMR === 'false'
 
+if (process.env.PWA_KIT_SSR_SOURCE_MAP) {
```

Comment:
It appears to me that we haven't mentioned about this env any where. I am putting a warning here to warn when it is used, we can remove when we start to release new major version